DBContextApp/Controllers/BreedsController.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using DBContextApp.Context;
+using DBContextApp.Models;
+using DBContextApp.Repository;
+
+namespace DBContextApp.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class BreedsController : ControllerBase
+    {
+        private readonly DataContext _context;
+
+        public BreedsController(DataContext context)
+        {
+            _context = context;
```

Comment:
Why are you accessing dbcontext here? What is the point of your repository then?